C#/C#.sublime-syntax

```diff
@@ -446,12 +446,7 @@ contexts:
     - match: \s*(;)
       captures:
         1: punctuation.terminator.statement.cs
-      pop: true
-
-  maybe_terminator:
-    - include: terminator
-    - match: (?=\S)
-      pop: true
+      pop: 1
```

Comment:
Not important, but you use `pop: true` for the rest of the commit (except the `pop: 2`).